packages/components/src/internal/components/settings/AuditSettings.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { FC, useCallback, useEffect, useState } from 'react';
+
+import { PermissionTypes } from '@labkey/api';
+
+import { ComponentsAPIWrapper, getDefaultAPIWrapper } from '../../APIWrapper';
+
+import { RequiresPermission } from '../base/Permissions';
+import { LabelHelpTip } from '../base/LabelHelpTip';
+import {
+    isAssayEnabled,
+    isDataChangeCommentRequirementFeatureEnabled,
+    isELNEnabled,
+    isWorkflowEnabled,
+} from '../../app/utils';
+import { useServerContext } from '../base/ServerContext';
+
+interface AuditSettingProps {
+    api?: ComponentsAPIWrapper;
+}
+
+export const AuditSettings: FC<AuditSettingProps> = props => {
+    const { api } = props;
+    const { container, moduleContext } = useServerContext();
+    const [isRequired, setIsRequired] = useState<string>();
```

Comment:
This should be a `boolean` and if not then I'd recommend leaving a comment explaining why not. Seems like the usages are all trying to work around this fact.

Reply:
Was a string because that's the type the radio buttons use, but I've since changed the callbacks, so, yeah, boolean is better here.